frontend/static/css/index.css

```diff
@@ -0,0 +1,119 @@
+body {
+    height: 100%;
+    margin: 0;
+  }
+
+.topnav {
+    /* overflow: hidden; */
+    /* background-color: rgb(148, 148, 148); */
+    background-color: #3b694483;
+    position: fixed; /* Set the navbar to fixed position */
+    top: 0; /* Position the navbar at the top of the page */
+    left: 0;
+    width: 100%; /* Full width */
+    z-index:1000;/*add z-index*/
+}
+
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 20px;
+}
+
+.topnav a.active {
+    background-color: #8cee8786;
+    color: rgb(92, 97, 54);
+}
+
+.topnav .icon {
+    display: none;
+}
+
+/* The dropdown container */
+.dropdown {
+    float: left;
+    overflow: hidden;
+}
+  
+/* Dropdown button */
+.dropdown .dropbtn {
+    font-size: 20px;
+    border: none;
+    outline: none;
+    color: white;
+    padding: 14px 16px;
+    background-color: inherit;
+    font-family: inherit; /* Important for vertical align on mobile phones */
+    margin: 0; /* Important for vertical align on mobile phones */
+    /* z-index:1000; */
+}
+
+  /* Show the dropdown menu on hover */
+.topnav a:hover, .dropdown:hover, .dropbtn:hover{
+    color: rgb(92, 97, 54);
+    background-color: #9ad8a58c;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: inherit;
+    min-width: auto;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+}
+/* Links inside the dropdown */
+.dropdown-content a {
+    float: none;
+    display: block;
+    color: rgb(255, 255, 255);
+    padding: 12px 16px;
+    text-decoration: none;
+    text-align: left;
+    z-index: 1000;
+}
+.dropdown:hover .dropdown-content{
```

Comment:
Here since we have no comma separating the elements, we can read it as:
When dropdown is hovered, the dropdown-content element will assume the properties below.